transcom/mymove
# MB-5790 replace usage of here planner with ghc planner for required delivery date

## [Jira ticket](https://dp3.atlassian.net/browse/MB-5970)

## Summary

This PR removes the obsolete HERE planner which was used to calculate the **Required Delivery Date** for a shipment. 

What was removed:
-  HERE planner removed from the server routing initialization during server startup
- The environment secrets related to HERE planner endpoint and credentials were deleted
- HERE planner removed from handlers config
- Replaced Here planner for HHG planner in the Prime api 
- Removed the mocks for `Planner.TransitDistance()` and replaced with `HHGPlanner.ZipTransitDistance()`
- Replaced Here planner for DTOD planner in the internal api (this does not get used)

## Verification Steps for Author and Reviewers

- Login in as the TOO
- Select an approved move with HHG Shipments and copy the move code
- Sign out and login in as prime simulator user
- Search for the copied move code
- Update the shipment to a future **Scheduled and Requested Pickup Date**
- Make sure to set/check **estimate weight** 
- Log out and sign in as TOO user
- Navigate to move details tab, and check that the **Required Delivery Date** is set and a future date

## Screen Capture

https://github.com/transcom/mymove/assets/111006369/cc8204c0-ad9e-4ac6-9c6d-fa4e1036e937
